src/main/java/com/parsing/model/LotPDFResult.java

```diff
@@ -34,7 +33,7 @@ public class LotPDFResult {
 
     @Id
     @GeneratedValue(generator = "UUID")
-    @Column(name ="id")
+    @Column(name = "id")
```

Comment:
Good job. (Just a positive comment so we are not looking like judgemental bastards)

Reply:
excellent comment)